flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
if you are building from source, why restrict to x86_64?

Reply:
Currently, I don't have the ability to test for other platforms. 
With a few changes to the build script, I could create a build (e.g. für arm64). However, I don't feel comfortable releasing it without testing it.

Reply:
if you don't release it nobody will test it. And it will stay that way forever, that I'm certain because it's never revisited.

Reply:
what's the point of bloating everything with electron to be "portable" and restricting architecture like it's 1995.

Reply:
I understand that this isn't ideal. I, too, would like Looksyk to work on other architectures.

I'm personally skeptical of the suggestion to simply test it in production. If that's what Flatpak wants, I can implement it.

Generally, the project is a hobby of mine, and I have limited resources. Since supporting different platforms is very time-consuming, I've currently focused on x84-64, even though I've had Arm64 support in mind for some time.

Reply:
It's electron + javascript, It's suppsoed to be portable.

And you think we are paid to do this, think again.